src/clusterfuzz/_internal/tests/core/bot/tasks/trials_test.py

```diff
@@ -40,6 +41,16 @@ def setUp(self):
 
     test_helpers.patch(self, ['random.random'])
 
+  def source_side_setup(self, config_file_content):
+    environment.set_value('SOURCE_SIDE_TRIALS', 'trials_config.json')
+    environment.set_value('APP_DIR', '/src')
+
+    test_utils.set_up_pyfakefs(self)
+    self.fs.create_file('/src/trials_config.json')
```

Comment:
In case we decide for always looking for a json file, then all tests need to use fake file system and we need tests with and without the file.

Reply:
done, I have added `environment.set_value('APP_DIR', '/src')` to the general setup